src/platform/shared.c

```diff
@@ -712,6 +712,10 @@ void platform_process_messages()
 				if (e.text.text[0] == '`' && gConsoleOpen)
 					break;
 
+				// Entering formatting characters is not allowed
+				if (utf8_is_format_code((int)e.text.text[0]))
```

Comment:
Yea, UTF-8 is backwards compatible with ASCII, its only because some format codes are not ASCII that we need to do this.